### DISCUSSION TITLE
Error when launching with VLC as default video player

### DISCUSSION THREAD

**Discussion by frenchfilm** [SubtitleEdit/subtitleedit#6254] [Q&A]
Hello
When I launch Subtitleedit (tried it with v3.3.8, 3.6.0 and new v3.6.7) with VLC set as default video player, the app crashes. Is it a codec problem (H265)?

![image](https://user-images.githubusercontent.com/39890330/190279240-ff4479d9-5fae-4fe1-874b-81d8ea799234.png)

**Comment by niksedk** [comment]
What OS are you using?
What version of VLC are you using?

How did you install SE? Portable version or Installer? To what folder was SE installed to?
